packages/electron/src/electron.ts

```diff
@@ -188,6 +188,22 @@ function createWindow() {
   autoUpdater.on('update-available', (info) => {
     latestVersion = info.version;
     outdatedVersionFlag = true;
+    dialog
+      .showMessageBox(mainWindow!, {
+        type: "info",
+        buttons: ["Download Now", "Later"],
+        title: "Application Update",
+        message:
+          "A new version is available. Click the \"Download Now\" button to update. You may also choose to do this later.",
```

Comment:
do we need to ask someone to verify the message?

Reply:
Verified the message with Sunanda.